pypeit/metadata.py

```diff
@@ -1576,7 +1580,7 @@ def set_user_added_columns(self):
         if 'manual' not in self.keys():
             self['manual'] = ''
         if 'shift' not in self.keys():
-            self['shift'] = 0
+            self['shift'] = self.MASKED_VALUE
```

Comment:
I'm a little worried about this approach, but I don't have a better solution off the top of my head.

Reply:
Not a fan of `-9999` in the pypeit file either. It seems to me that `0` would work in the same way, unless (I guess) somebody want to force a 0 shift...

Reply:
That's correct - if someone sets `manual_shift=0`, then the manual shift wouldn't be applied, because the `manual_shift` is set to the (hard-coded) masked value.

I am also not a huge fan, but I don't have a better solution. Although, I thought the `-9999` wouldn't appear in the `.pypeit` file, it's only something that's hidden in the code. Am I mistaken about this?

Reply:
The -9999 would appear in the `.pypeit` file if you run `pypeit_setup -f` (the `shift` column will be added). Currently that column would have all the values to 0 and then the user would change those values as they prefer. So technically, that would work the same with -9999, but I would prefer to not see -9999 in the `.pypeit` file, if possible. I wonder if we could just set the default to `None`? There are other metadata that are sometime `None`, so I don't think it should be a problem. In addition, the condition in `pypeit.py` would catch it and not use it anyway.

What do you guys think? @rcooke-ast @kbwestfall

Reply:
OK - I think I have a better solution -- let me know what you think!  Previously, we were getting rid of masked columns. So, if we reinstate masked columns in the `fitstbl`, then we can simply use masked values. This means that the `shift` column of the pypeit file will just be empty for masked values, and spatial flexure would only be applied to frames with non-empty (i.e. non-masked) values. I don't think this has any undesired effects... 😬